packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -914,39 +912,8 @@ function ResizableTableColumnHeader(props) {
     return options;
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [allowsSorting]);
-  let isMobile = useIsMobileDevice();
 
   let resizingColumn = layout.resizingColumn;
-  let prevResizingColumn = useRef(null);
-  let timeout = useRef(null);
-  useEffect(() => {
-    if (prevResizingColumn.current !== resizingColumn &&
-      resizingColumn != null &&
-      resizingColumn === column.key) {
-      if (timeout.current) {
-        clearTimeout(timeout.current);
-      }
-      // focusSafely won't actually focus because the focus moves from the menuitem to the body during the after transition wait
-      // without the immediate timeout, Android Chrome doesn't move focus to the resizer
-      let focusResizer = () => {
-        resizingRef.current.focus();
-        onFocusedResizer();
-        timeout.current = null;
-      };
-      if (isMobile) {
-        timeout.current = setTimeout(focusResizer, 400);
-        return;
-      }
-      timeout.current = setTimeout(focusResizer, 0);
-    }
-    prevResizingColumn.current = resizingColumn;
-  }, [resizingColumn, column.key, isMobile, onFocusedResizer, resizingRef, prevResizingColumn, timeout]);
-
-  // eslint-disable-next-line arrow-body-style
-  useEffect(() => {
-    return () => clearTimeout(timeout.current);
-  }, []);
-
```

Comment:
This is handled in the hook now. The onFocusedResizer moved to the onFocus event of the resizer itself.